contracts/Marketplace.sol

```diff
@@ -1,74 +1,296 @@
-pragma solidity ^0.4.24;
-
-import "openzeppelin-solidity/contracts/utils/Address.sol";
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
-import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
-
-contract Service is Ownable, Pausable {
-    bytes32 public sid;
-    bytes32 public latest;
-    mapping (bytes32 => Version) public versions;
-    mapping (bytes32 => mapping(address => bool)) private _versionAccess;
-
-    struct Version {
-        bytes32 id;
-        bytes32 location;
-        uint256 price;
-        bool _exists;
-    }
+pragma solidity >=0.5.0 <0.6.0;
+
+import "../node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
why used `../node_modules/` instead of let truffle do discovery?

Reply:
Truffle is compatible and it solves imports error with vsc solidity linter